vars/odsComponentStageScanWithSnyk.groovy

```diff
@@ -0,0 +1,15 @@
+import org.ods.component.ScanWithSnykStage
+import org.ods.component.IContext
+
+import org.ods.services.SnykService
+import org.ods.services.ServiceRegistry
+
+def call(IContext context, Map config = [:]) {
+    def snykService = ServiceRegistry.instance.get(SnykService)
+    if (!snykService) {
+        snykService = new SnykService('snyk-report.txt')
```

Comment:
there is no constructor for this, you probably mean:
`snykService = new SnykService(this, 'snyk-report.txt')`
?

Reply:
Fixed in aac6b07.